affinity/cpp-23/d1436r3.md

```diff
@@ -469,7 +469,7 @@ There are a number of additional features which we are considering for inclusion
 
 ## Iteration space subdivision property
 
-It is defined in this proposal for the `bulk_execution_affinity_t` properties that when the size in an invocation of `execution::bulk_execute` is greater than the *available concurrency* then it is implementation how that iteration space is subdivided into a consecutive sequence of work-items. The authors of this proposal intend to propose a follow up property for specifying how an iteration space should be subdivided into chunks in this case.
+It is defined in this proposal for the `bulk_execution_affinity_t` properties that when the size in an invocation of `execution::bulk_execute` is greater than the *available concurrency* tthen it is implementation defined how that iteration space is subdivided into a consecutive sequence of work-items. The authors of this proposal intend to propose a follow up property for specifying how an iteration space should be subdivided into chunks in this case.
```

Comment:
"tthen" (doubled letter) -- not sure if you can upload a fix still but no biggie :-)

Reply:
Oh no how did I miss that :(

Thanks, yeah I should be able to update it before the mailing goes out :)

Reply:
I've updated the paper, thanks again :)